.gitignore

```diff
@@ -3,5 +3,5 @@ public/lib
 .sass-cache/
 .DS_store
 c4h_keys.txt
-.env
+*.env
```

Comment:
The * was to use a wildcard to ensure that files ending with .env don't get tracked by git. However, on second thought, I think there can only be 1 .env file so the .env is not wrong either